karaage/common/passwords.py

```diff
@@ -48,6 +49,15 @@ def assert_strong_password(username, password, old_password=None):
     """Raises ValueError if the password isn't strong.
 
     Returns the password otherwise."""
+
+    # test the length
+    try:
+        minlength = settings.MIN_PASSWORD_LENGTH
+    except AttributeError:
+        minlength = 12
```

Comment:
If it's a problem that the default has been set to 12 here, we're happy to make this 8 or any other value and just use the new MIN_PASSWORD_LENGTH setting to set to 12 in our deployments.